### PR TITLE
Add WebRender components

### PR DESCRIPTION
**Pull request by staktrace**

### CODE REVIEW

File: src/data/gfx.yaml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 name: Graphics
 summary: Get involved with the [Graphics](https://wiki.mozilla.org/Platform/GFX/Contribute) team
 products: 
- - Core: ['Graphics', 'GFX: Color Management', 'Canvas: WebGL', 'Canvas: 2D', 'ImageLib', 'Graphics', 'Graphics: Layers', 'Graphics: Text', 'Panning and Zooming']
+ - Core: ['Graphics', 'GFX: Color Management', 'Canvas: WebGL', 'Canvas: 2D', 'ImageLib', 'Graphics', 'Graphics: Layers', 'Graphics: Text', 'Graphics: WebRender', 'Panning and Zooming']
+repositories:
+ - servo/webrender: WebRender
```
[the remark this answers is not in the document]
Comment (staktrace) [reply src/data/gfx.yaml]: What happens if multiple labels are specified? Does that match issues with any of the labels, or all of the labels?

[the remark this answers is not in the document]
Comment (fiennyangeln) [reply src/data/gfx.yaml]: any of them.

**Comment by staktrace** [comment]
Thanks. PR updated ^

File: src/data/gfx.yaml [hunk 2]
```diff
@@ -1,4 +1,6 @@
 name: Graphics
 summary: Get involved with the [Graphics](https://wiki.mozilla.org/Platform/GFX/Contribute) team
 products: 
- - Core: ['Graphics', 'GFX: Color Management', 'Canvas: WebGL', 'Canvas: 2D', 'ImageLib', 'Graphics', 'Graphics: Layers', 'Graphics: Text', 'Panning and Zooming']
+ - Core: ['Graphics', 'GFX: Color Management', 'Canvas: WebGL', 'Canvas: 2D', 'ImageLib', 'Graphics', 'Graphics: Layers', 'Graphics: Text', 'Graphics: WebRender', 'Panning and Zooming']
+repositories:
+ - servo/webrender: help-wanted, difficulty: easy
```
Comment (fiennyangeln) [inline src/data/gfx.yaml]: I think it should be `['help-wanted', 'difficulty: easy']`  to be parsed as different string

Comment (staktrace) [reply src/data/gfx.yaml]: Oops, PR updated again to fix that

**Comment by staktrace** [comment]
Also /cc @eqrion since he's reworking a bunch of the gfx wiki pages and might want to flesh out the introduction here once he's done that and has a good landing page for potential new contributors.

**Comment by fiennyangeln** [comment]
@helfi92 wdyt of this change?

**Comment by staktrace** [comment]
Done

**Pull request merged**